src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs

```diff
@@ -23,14 +23,14 @@ public class GitFileManager
         private readonly ILogger _logger;
 
         // Matches package feeds like
-        // https://dnceng.pkgs.visualstudio.com/public/_packaging/darc-pub-arcade-fd8184c3fcde81eb27ca4c061c6e171f418d753f-1
+        // https://dnceng.pkgs.visualstudio.com/public/_packaging/darc-pub-arcade-fd8184c3fcde81eb27ca4c061c6e171f418d753f-1/nuget/v3/index.json
```

Comment:
I guess you'd want to remove this as well?

Reply:
AFAIK, the Azure package feeds do include the nuget/v3 part.

@jcagme for confirmation

Reply:
Correct `https://<org>.pkgs.visualstudio.com/_packaging/<feed-name>/nuget/v3/index.json`